xml/vk.xml

```diff
@@ -7627,9 +7627,9 @@ typedef void* <name>MTLSharedEvent_id</name>;
             <member><type>uint32_t</type>               <name>blockDimZ</name></member>
             <member><type>uint32_t</type>               <name>sharedMemBytes</name></member>
             <member optional="true"><type>size_t</type>                 <name>paramCount</name></member>
-            <member optional="true" len="paramCount">const <type>void</type>* const *    <name>pParams</name></member>
+            <member len="paramCount">const <type>void</type>* const *   <name>pParams</name></member>
```

Comment:
Now there is no information regarding the inner pointer. It is unclear to me whether these are allowed to be null or not, but in any case, I would expect either `optional="false,true"` or `optional="false,false"` here.

Reply:
yes, that would be better. Not having an `optional` attribute is definitely the exception. According to the referenced documentation here https://docs.nvidia.com/cuda/cuda-driver-api/group__CUDA__EXEC.html#group__CUDA__EXEC_1gb8f3dc3031b40da29d5f9a7139e52e15 it seems as though this ought to be `false,false`.

Reply:
removing  optional=true for pParam sounds like the right choice.

Reply:
Note: I have the feeling the same `optional="true"` should be removed from  
`<member optional="true" len="extraCount">const <type>void</type>* const *    <name>pExtras</name></member>`